audit-service/src/main/java/uk/gov/gchq/palisade/service/audit/service/StroomAuditService.java

```diff
@@ -213,19 +223,34 @@ public StroomAuditService systemName(final String systemName) {
         return this;
     }
 
+    /**
+     * Returns the system name
+     *
+     * @return the system name
+     */
     @Generated
     public String getSystemName() {
         return SYSTEM.getName();
     }
 
+    /**
+     * Sets the new system name from which the audit service is receiving audit logs
+     * from
+     *
+     * @param systemName the name of the system
+     * @throws NullPointerException is {@code systemName} is null
```

Comment:
```suggestion
     * @throws NullPointerException if {@code systemName} is null
```
The same change should also be made on lines 253, 278, 291, 315, 328, 353, 365, 388, 401 and 429